### DISCUSSION TITLE
No documentation about use of the REST API for HPOS orders

### DISCUSSION THREAD

**Discussion by evroon** [woocommerce/woocommerce#37490] [WooCommerce HPOS] [answered]
There seems to be no documentation about how the REST API can be used for HPOS orders. I looked in the following places:
- I can't find anything mentioning `HPOS` in the API docs: https://woocommerce.github.io/woocommerce-rest-api-docs/#. 
- The [HPOS blog post](https://developer.woocommerce.com/2022/09/14/high-performance-order-storage-progress-report/) says there should have been an update about `UI and API changes`, but it doesn't seem to exist
- The [technical document](https://woocommerce.com/document/high-performance-order-storage/) also doesn't mention the REST API.
- The [FAQs](https://developer.woocommerce.com/2022/10/11/hpos-upgrade-faqs/) also don't

So how does the REST API relate to HPOS orders, can the same endpoints be used as usual with no difference between HPOS and traditional orders or are there breaking changes?

**Comment by vedanshujain** [comment, marked as answer]
Hey @evroon , you can use the same orders endpoint without any difference for both HPOS and traditional orders. There shouldn't be any breaking changes.

**Reply by evroon** [reply]
Thanks!

**Comment by thejamescollins** [comment]
As far as I can tell, the V2 and V3 orders endpoints are compatible with HPOS, but the V1 orders endpoint is not.
[Reference](https://woocommercecommunity.slack.com/archives/C043X91E72M/p1688435654974489).
